src/components/settings/SettingsLMSTab/tests/BlackboardConfig.test.jsx

```diff
@@ -97,34 +101,21 @@ describe('<BlackboardConfig />', () => {
     };
     expect(LmsApiService.updateBlackboardConfig).toHaveBeenCalledWith(expectedConfig, 1);
   });
-  test('it creates new configs on submit', () => {
```

Comment:
I know they're accomplishing kinda the same thing, but we should probably keep the test of the submitting a new config as well as have this new test, or am I crazy?